CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-24.01+ (???)
+24.01.1 (???)
 ------------------------------------------------------------------------
+- Fix: [#2253]: Fixes game crashing when loading some save games.
```

Comment:
Sorry, one more thing. The convention is to state the original issue here rather than the fix, e.g.:
```suggestion
- Fix: [#2253]: Game crashes when loading some save games on Windows.
```